src/Provider/Zendesk.php

```diff
@@ -81,7 +83,24 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
      */
     protected function getDefaultScopes()
     {
-        return [];
+        return $this->scopes;
+    }
+
+    /**
+     * Set the default scopes used by this provider.
+     *
+     * This should not be a complete list of all scopes, but the minimum
+     * required for the provider user interface!
+     *
+     * @return Zendesk
+     */
+    public function setDefaultScopes(Array $scopes)
```

Comment:
@chadhutchins As an FYI, this line was responsible for the CI failure...

```
----------------------------------------------------------------------
549FOUND 1 ERROR AFFECTING 1 LINE
550----------------------------------------------------------------------
551 97 | ERROR | [x] PHP keywords must be lowercase; expected "array"
552    |       |     but found "Array"
553----------------------------------------------------------------------
554PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
555----------------------------------------------------------------------
```

There is a code style linter that runs as a part of the build.